nextroute/model_constraint_maximum_wait_stop.go

```diff
@@ -12,9 +14,11 @@ type MaximumWaitStopConstraint interface {
 	Maximum() StopDurationExpression
 }
 
-// NewMaximumWaitStopConstraint returns a new MaximumWaitStopConstraint. The maximum
-// wait constraint limits the time a vehicle can wait between two stops. Wait
-// time is defined as the time a vehicle is neither moving nor serving a stop.
+// NewMaximumWaitStopConstraint returns a new MaximumWaitStopConstraint. The
+// maximum wait constraint limits the time a vehicle can wait between two stops.
+// Wait is defined as the time between starting a stop and arriving at the
+// location of the stop, [SolutionStop.StartValue()] -
```

Comment:
Same here

Reply:
If you start at the time you arrive there is no waiting, starting later than arriving creates waiting time (at the gate for example).